tools/valgrind.supp

```diff
@@ -10,3 +10,16 @@
    fun:cockpit_certificate_find
    fun:main
 }
+
+# Valgrind reports this on clang/i386 and the backtrace does not include our
+# own code
+{
+   nettle_conditional_jump
+   Memcheck:Cond
+   fun:_nettle_ecc_pp1_redc
```

Comment:
looks like you need a more generic rule

probably just mask out everything with `fun:nettle_ecc_point_mul` anywhere in the trace

Reply:
```
==3589== More than 1000 different errors detected.  I'm not reporting any more.
==3589== Final error counts will be inaccurate.  Go fix your program!
```

This seems hopeless :-)